package.json

```diff
@@ -85,7 +85,7 @@
     "d3-queue": "^3.0.7",
     "del": "^3.0.0",
     "eslint": "~4.14.0",
-    "eslint-config-stylelint": "^8.0.0",
+    "eslint-config-stylelint": "^8.1.0",
```

Comment:
I think we should pin it as well as ESLint: 

```
"eslint-config-stylelint": "~8.1.0",
```

Reply:
Done